core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestUtils.java

```diff
@@ -76,10 +78,16 @@
    * @return the response
    */
   private static Response createResponse(Object object) {
-    if (object instanceof Void) {
+    if (object == null || object instanceof Void) {
```

Comment:
does `object instanceof Void` do anything?

Reply:
done, removed in another PR.